v8/mastodoncomments/conf.py.sample

```diff
@@ -0,0 +1,10 @@
+COMMENT_SYSTEM = "MastodonComments"
+COMMENT_SYSTEM_ID = "<YOUR-SITE_NAME>"
```

Comment:
Is this needed/used?

Reply:
Actually, I didn't know what its original purpose was. I now read the Nikola documentation and found out that the COMMENT_SYSTEM_ID is used by several comment systems for specifying something like the next step in the process of accessing the comments.

In the light of this I will remove mastodon_config['host'] and use comment_system_id in its place. Until there's a bigger redesign in how to configure the comment systems, e.g. in a designated dict, I think its best to stick to lived practice.